aptos-move/e2e-testsuite/src/tests/genesis.rs

```diff
@@ -50,3 +49,18 @@ fn execute_genesis_and_drop_other_transaction() {
     assert_eq!(output.len(), 2);
     assert_eq!(output.pop().unwrap().status(), &TransactionStatus::Retry)
 }
+
+#[test]
+fn fail_no_epoch_change_write_set() {
+    let mut executor = FakeExecutor::no_genesis();
+    let txn = Transaction::GenesisTransaction(WriteSetPayload::Direct(ChangeSet::empty()));
+
+    let sender = executor.create_raw_account_data(1_000_000, 10);
+    let receiver = executor.create_raw_account_data(100_000, 10);
+    let txn2 = peer_to_peer_txn(sender.account(), receiver.account(), 11, 1000, 0);
+
+    let output_err = executor
+        .execute_transaction_block(vec![txn, Transaction::UserTransaction(txn2)])
```

Comment:
Will it work if it's just txn?

Reply:
that's the test above - execute_genesis_write_set ?